src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -284,6 +286,13 @@ const AdvancedSettings: React.FC<AdvancedSettingsProps> = ({
 
         {renderShowGridToggle()}
 
+        <ToggleElement
+          forwardRef={register}
+          isChecked={values.type}
+          name="type"
+          title="Set room un-clickable"
+        />
```

Comment:
The unclickable value isn't part of the `Venue`, it should be part of the `Room` (and is already defined on that type):

- https://github.com/sparkletown/sparkle/blob/staging/src/types/rooms.ts#L9
- https://github.com/sparkletown/sparkle/blob/staging/src/types/rooms.ts#L28

This shouldn't be a `boolean`, it is an enum of `RoomTypes`.

However, we could choose to 'hide' that nuance in the frontend for now if we want by making it a 'checkbox' that will cause the enum value to be set. This might be simpler UI/UX for now, as we only have a single `type` at the moment.